.github/labeler.yaml

```diff
@@ -6,9 +6,8 @@ ci:
   - .clang-format
   - .gitignore
   - .prettierignore
-document:
+documentation:
```

Comment:
If you add the `documentation` label, the `deploy-docs` workflow may run, which might become a security hole.
Do you think it's okay?

Reply:
If you want to use the `documentation` label, I believe it's better to use a different `label` for `deploy-docs` to avoid unnecessary risks.
https://github.com/autowarefoundation/autoware.universe/blob/7e37a6c65a6a72ddf537ac673bf76cd3ff7ae581/.github/workflows/deploy-docs.yaml#L24

Reply:
> If you want to use the documentation label, I believe it's better to use a different label for deploy-docs to avoid unnecessary risks.

Done by https://github.com/autowarefoundation/autoware.universe/pull/2302#pullrequestreview-1181967226